stisim/diseases/hiv_main.py

```diff
@@ -0,0 +1,204 @@
+"""
+Run hiv
```

Comment:
What's the difference between this file and hiv.py?

Reply:
`hiv.py` is the disease module, `hiv.main.py` is what I added for running simulations with different parameters (for calibration).` hiv.main.py `basically contains the simulation setup like in `run_hiv.y` and is called by the celery-module to set up the simulation with specific input params. Does that make sense? I am not sure if it should live in the disease-folder. For the gavi work, we had these types of files living in what would be the syphilis-analyses->stisim folder

Reply:
I think this one would probably also be analysis-specific. The role of this file would be to create and configure a `Sim` for the analysis so I suspect this will eventually morph into something that sets up both the HIV and syphilis modules and associated interventions (so maybe it'll get renamed too)